src/domain/landingPage/landingPageHero/LandingPageHero.tsx

```diff
@@ -32,6 +60,58 @@ const LandingPageHero: React.FC<Props> = ({ landingPage }) => {
     window.open(buttonUrl || '', '_self');
   };
 
+  // We want to have event padding for the text wrapper. By using pure CSS we can set
+  // max-width for the wrapper but when the text is longer that max-width the text
+  // will take space of max-width and there might be wider right padding.
+  // So we need to calculate the max-width of each text row and set max-width of
+  // text wrapper to same
+  const calculateTextWrapperWidth = () => {
```

Comment:
I meant that this could be hooks outside of this component because it is so long :) I didn't mean useMemo is not needed

Reply:
That useMemo wasn't useful since title was changing on every render. But yeah I could write an external hook for this anyway.